### PR TITLE
feature/click

### PR DESCRIPTION
**Pull request by NicolasGradiweb**
# **Why are these changes introduced?**

This is done in order to handle text appearing with a button click

**Other considerations**
**Demo links**

- [Store](Store)
- [Editor](Editor)

  **Checklist**

- [ ] Followed [theme code principles](https://airtable.com/apph9XWekpWF7EEwF/tblzvrsI7MvSyhWRS/viwNghgN1cVW1CYIO?blocks=hide)
- [ ] Linted with [Theme Check](https://github.com/Shopify/theme-check)
- [ ] Tested on [mobile](https://shopify.dev/themes/store/requirements#mobile-browser-requirements)
- [ ] Tested on [multiple browsers](https://shopify.dev/themes/store/requirements#desktop-browser-requirements)
- [ ] Tested for [accessibility](https://shopify.dev/themes/best-practices/accessibility)
      **_Purchase flow (can be product page or customized pages) If applicable_**
- [ ] Images are displayed correctly.
- [ ] The product options are correctly represented.
- [ ] The user can choose a combination of options.
- [ ] Interactions between choice of options and images (see [Bonsoirs](https://bonsoirs.com/) as an example) work correctly.
- [ ] The variant can be successfully added to the cart.
- [ ] The variant added to the cart corresponds to the previously selected combination of options.
      **_Cart (page and mini-cart if applicable)_**
- [ ] All the products that are added to the cart are displayed.
- [ ] Product information is correct and complete.
- [ ] The images are displayed correctly and correspond to the variants in the cart.
- [ ] The price in the cart corresponds to the price of the variant.
- [ ] The inputs to modify the _line_items_ work correctly.
- [ ] The up-sell/cross-sell works correctly.
      **_Debbugging - All debugging steps have been removed/commented out:_**
- [ ] Alerts
- [ ] Console logs
- [ ] Texts

### CODE REVIEW

File: main.css [hunk 1]
```diff
@@ -7,4 +7,6 @@
 
 .text {
   font-size: 1.25rem;
-}
\ No newline at end of file
+}
+
+.hidden {display: none;}
```
Comment (Nimoscmp) [inline main.css]: Por favor organiza las propiedades en líneas separadas

File: main.js [hunk 2]
```diff
@@ -0,0 +1,12 @@
+const a = () => {
```
Comment (Nimoscmp) [inline main.js]: Recuerda poner un nombre claro para la función

File: main.js [hunk 3]
```diff
@@ -0,0 +1,12 @@
...
+  var button = document.querySelector('.button');
```
Comment (Nimoscmp) [inline main.js]: No se siguen los lineamientos de la biblia técnica. Prohibido usar var.

**Review by Nimoscmp — CHANGES_REQUESTED** [review]
Por favor revisa los comentarios

File: main.css (hunk 1 above)
Comment (NicolasGradiweb) [reply main.css]: Ya está corregido

File: main.js (hunk 2 above)
Comment (NicolasGradiweb) [reply main.js]: Se cambió el nombre de la función

File: main.js (hunk 3 above)
Comment (NicolasGradiweb) [reply main.js]: Ya se reemplazó var por const

**Pull request merged**